### PR TITLE
Feature/json server fixtures

### PR DESCRIPTION
**Pull request by tuj**
AR-642
* Added json-server setup based on previous frontend fixtures.
* Changed pull-strategy to match API.
* Removed public/fixtures/*.json files.

Work left for coming PRs:

- Move json-server setup into its own repo to make it available for admin as well.
- Handle pagination

TODOs:

- [x] Fix names of screen fixtures
- [x] Fix nginx setup to avoid blocking :3000
- [x] Fix tests

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -62,9 +62,12 @@ jobs:
       - name: Setup network
         run: docker network create frontend
 
-      - name: Install
+      - name: Install client
         run: docker-compose run node yarn
 
+      - name: Install and start json-server
```
Comment (cableman) [inline .github/workflows/pr.yml:68]: How do that start the json-server? Will it not shutdown when install is completed?

File: data/app.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const jsonServer = require('json-server');
+
+const server = jsonServer.create();
+const middlewares = jsonServer.defaults();
+const v1Router = jsonServer.router('v1.json');
+
+const hydraRender = (req, res) => {
+  let { data } = res.locals;
+
+  if (req.originalUrl.startsWith('/v1/playlistScreenRegion')) {
```
Comment (cableman) [inline data/app.js:10]: Is this end-point in the API design?

File: data/app.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    data = data.map((playlistScreenRegion) => playlistScreenRegion.playlist);
+  }
+
+  if (req.originalUrl.startsWith('/v1/slidesPlaylist')) {
```
Comment (cableman) [inline data/app.js:14]: Is this end-point in the API design?

File: src/debug-bar.js [hunk 4]
```diff
@@ -10,199 +9,24 @@ import { v4 as uuidv4 } from 'uuid';
...
-   * @param {boolean} enabled
-   *   Boolean. Enabled or disable data synchronization.
-   */
-  function setDataSync(enabled) {
-    if (enabled) {
-      document.dispatchEvent(new Event('startDataSync'));
-    } else {
-      document.dispatchEvent(new Event('stopDataSync'));
-    }
+    setTimeout(() => document.dispatchEvent(new Event('stopDataSync')), 1000);
```
Comment (cableman) [inline src/debug-bar.js:29]: Timeout?

**Review by cableman — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yml (hunk 1 above)
Comment (tuj) [reply .github/workflows/pr.yml:68]: Fixed

File: data/app.js (hunk 2 above)
Comment (tuj) [reply data/app.js:10]: Its a hack for handling the relations. This is the url that is in the mock api json. I'll add a comment about it

File: src/debug-bar.js (hunk 4 above)
Comment (tuj) [reply src/debug-bar.js:29]: This is just to stop the sync calls, so it only runs one time when the fixture is loaded.

File: data/app.js (hunk 3 above)
Comment (tuj) [reply data/app.js:16]: Added comment

File: src/debug-bar.js (hunk 4 above)
Comment (tuj) [reply src/debug-bar.js]: Removed it

**Review by cableman — APPROVED** [review]

**Pull request merged**